packages/ckb-sdk-rpc/src/defaultRPC.ts

```diff
@@ -172,7 +172,7 @@ export class DefaultRPC {
    * @param {string} number - the block number of the target block
    * @returns {Promise<object>} block object
    */
-  public getBlockByNumber!: (number: CKBComponents.BlockNumber) => Promise<CKBComponents.Block>
+  public getBlockByNumber!: (number: CKBComponents.BlockNumber | number) => Promise<CKBComponents.Block>
```

Comment:
So we're having this rule for number param: it's either hex string, or decimal number? (Sorry if we've already describe that rule in other places.)

Reply:
> So we're having this rule for number param: it's either hex string, or decimal number? (Sorry if we've already describe that rule in other places.)

Yes, it allows integer like `123` and hex string like `0x123` while integer in string format `'123'` is forbidden.

Reply:
Nice to know that. Thanks.